core/build.gradle

```diff
@@ -152,6 +152,13 @@ dependencies {
     //Play In-App Review
     api "com.google.android.play:review-ktx:$in_app_review"
 
+    // Segment Library
+    api "com.segment.analytics.kotlin:android:1.14.2"
+    // Segment's Firebase integration
+    api 'com.segment.analytics.kotlin.destinations:firebase:1.5.2'
+    // Braze SDK Integration
+    api "com.braze:braze-segment-kotlin:1.4.2"
```

Comment:
These dependencies can be moved as `implementation` dependencies to the app module since they are never used in other modules.

Reply:
moved.